javatests/arcs/core/host/ArcHostContextParticleTest.kt

```diff
@@ -37,23 +36,18 @@ import org.junit.runners.JUnit4
 class ArcHostContextParticleTest {
   private lateinit var schedulerProvider: SimpleSchedulerProvider
   private lateinit var random: Random
-  private lateinit var particles: MutableList<ArcHostContextParticle>
 
   @Before
-  fun setUp() = runBlocking {
-    schedulerProvider = SimpleSchedulerProvider(
-      Executors.newSingleThreadExecutor().asCoroutineDispatcher()
-    )
+  fun setUp() {
+    schedulerProvider = SimpleSchedulerProvider(Dispatchers.Default)
     random = Random(System.currentTimeMillis())
-    particles = mutableListOf()
   }
 
   @After
   fun tearDown() = runBlocking {
     schedulerProvider.cancelAll()
+    RamDisk.clear()
```

Comment:
did you revert the changes in this file on purpose, or did you not sync the latest changes?

Reply:
Ah, I may not have synced to the latest changes; I pulled this from another branch, which is likely stale.

Reply:
hmm... I pulled the changes with a `get checkout --` from another branch, so I'll need to do a manual revert on the relevant changes. Thanks for catching this.

Reply:
Should be fixed now.